platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureContainer.kt

```diff
@@ -72,6 +72,12 @@ class FeatureContainer(context: Context, attrs: AttributeSet? = null) :
 
     var errorHandler: ((String) -> Unit)? = null
 
+    // It appears WebMessagePort keeps weak references to message callbacks,
+    // which causes them to be garbage collected after a while, causing a
+    // communication breakdown between platform and feature.
+    // Therefore, this is a field.
```

Comment:
Figured a comment makes sense, because normally you wouldn't expect weak references in Java/Kotlin :D Still not sure if it's an actual weak reference or just a bug in `WebMessagePort`, but I'm trying to keep people from inlining this one :D

Reply:
Would it make more sense down there? Guess either location is noticeable enough.

Reply:
Keep this comment.